synapse/storage/schema/main/delta/91/01_media_hash.sql

```diff
@@ -0,0 +1,20 @@
+--
+-- This file is licensed under the Affero General Public License (AGPL) version 3.
+--
+-- Copyright (C) 2025 New Vector, Ltd
+--
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU Affero General Public License as
+-- published by the Free Software Foundation, either version 3 of the
+-- License, or (at your option) any later version.
+--
+-- See the GNU Affero General Public License for more details:
+-- <https://www.gnu.org/licenses/agpl-3.0.html>.
+
+-- Add a column `sha256` to `local_media_repository` table.
```

Comment:
comment is a little bit redundant for anyone who can read SQL

but maybe better said:

```suggestion
-- Store the SHA256 content hash of media files.
```